packages/metadata/src/metadata-service.ts

```diff
@@ -296,8 +296,8 @@ export class MetadataService implements IMetadataService {
     /**
      * Get Action or Function overloads by their top level name.
      *
-     * @param topLevelName
-     * @returns
+     * @param topLevelName string
+     * @returns string or undefined
```

Comment:
Avoid duplicating type information in descriptions which is already defined via type annotations.

Reply:
updated